examples/01_basic/fracture_analysis_contact_debonding.py

```diff
@@ -44,414 +45,622 @@
 
-MODEL.Activate()
-FRACTURE = MODEL.AddFracture()
 
-CONTACT_DEBONDING = FRACTURE.AddContactDebonding()
-CONTACT_DEBONDING.Material = MAT_CZM.Name
-CONTACT_DEBONDING.ContactRegion = CONTACT_REGION
+# %%
+# Add sizing to the mesh for the short and long edges
+add_sizing(mesh, "Short_Edges", Quantity("0.75 [mm]"), SizingBehavior.Hard)
+add_sizing(mesh, "Long_Edges", Quantity("0.5 [mm]"), SizingBehavior.Hard)
 
 # %%
-# Define analysis settings
-# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Add sizing to the mesh for both faces
+sizing_mesh_both_faces = mesh.AddFaceMeshing()
+sizing_mesh_both_faces.Location = get_child_object(
+    named_selections, Ansys.ACT.Automation.Mechanical.NamedSelection, "Both_Faces"
+)
+# Set the face meshing method to quadrilaterals
+sizing_mesh_both_faces.Method = FaceMeshingMethod.Quadrilaterals
 
-ANALYSIS_SETTINGS.Activate()
-ANALYSIS_SETTINGS.AutomaticTimeStepping = AutomaticTimeStepping.On
-ANALYSIS_SETTINGS.DefineBy = TimeStepDefineByType.Substeps
-ANALYSIS_SETTINGS.MaximumSubsteps = 100
-ANALYSIS_SETTINGS.InitialSubsteps = 100
-ANALYSIS_SETTINGS.MinimumSubsteps = 100
-ANALYSIS_SETTINGS.LargeDeflection = True
+# Activate and generate the mesh
+mesh.Activate()
+mesh.GenerateMesh()
+
+# Display the mesh image
+set_camera_and_display_image(
+    camera, graphics, graphics_image_export_settings, output_path, "mesh.png"
+)
+
+# %%
+# Add a contact debonding object
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Activate the model
+model.Activate()
+
+# Add a fracture to the model
+fracture = model.AddFracture()
+
+# Add contact debonding to the fracture
+contact_debonding = fracture.AddContactDebonding()
+# Set the material for the contact debonding
+contact_debonding.Material = get_child_object(
+    model_materials, Ansys.ACT.Automation.Mechanical.Material, "CZM Crack Material"
+).Name
+# Set the contact region for the contact debonding
+contact_debonding.ContactRegion = contact_region
+
+# %%
+# Define the static structural analysis settings
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Define the static structural analysis settings
+analysis_settings = static_structural_analysis.AnalysisSettings
+# Activate the analysis settings
+analysis_settings.Activate()
+# Turn on automatic time stepping
+analysis_settings.AutomaticTimeStepping = AutomaticTimeStepping.On
+# Define the time step settings with substeps
+analysis_settings.DefineBy = TimeStepDefineByType.Substeps
+# Set the initial, minimum, and maximum time step sizes
+analysis_settings.InitialSubsteps = 100
+analysis_settings.MinimumSubsteps = 100
+analysis_settings.MaximumSubsteps = 100
+# Turn on large deflection
+analysis_settings.LargeDeflection = True
 
 # %%
 # Define boundary conditions
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~
-# Add fixed support
 
-STATIC_STRUCTURAL.Activate()
-FIXED_SUPPORT = STATIC_STRUCTURAL.AddFixedSupport()
-FIXED_SUPPORT.Location = NS_EDGES_FIXED
+# Activate the static structural analysis
+static_structural_analysis.Activate()
```

Comment:
I'm not sure that all these Activate() functions are required. Please try to reduce to the minimum necessary for the script to work.